docs/src/main/tut/guidelines.md

```diff
@@ -69,6 +69,19 @@ The user doesn't need to specify the type `A` which is given by the parameter.
 You probably noticed that there is a `val dummy: Boolean` in the `PurePartiallyApplied` class. This is a trick we used
 to make this intermediate class a [Value Class](http://docs.scala-lang.org/overviews/core/value-classes.html) so that there is no cost of allocation, i.e. at runtime, it doesn't create an instance of `PurePartiallyApplied`. We also hide this partially applied class by making it package private and placing it inside an object.
 
+### <a id="ops-classes" href="#implicit-naming"></a> Implicit naming
 
-#### TODO: 
+In a widely-used library it's important to minimize the chance that the names of implicits will be used by others and
+therefore name our implicits according to the following rules:
+
+- Implicits should start with "cats" followed by the package name (where the instance is defined).
+- If the package contains `instances` leave `instances` out.
+- The type and the type class should be mentioned in the name.
+- If the instance is for multiple type classes, use `InstancesFor` instead of a type class name.
+- If the instance is for a standard library type add `Std` after the package. i.e. `catsStdShowForVector` and `catsKernelStdGroupForTuple`.
+
+As an example, for `cats.kernel.Monoid[List[A]]`, the name of the implicit should be `catsKernelMonoidForList`.
```

Comment:
This example may give the false impression that the package name is the package of the type class. How about 
```
As an example, an implicit instance of `Monoid` for `List` defined in the package `Kernel` should be named as `catsKernelMonoidForList`
```